source/numbers.rst

```diff
@@ -122,7 +123,9 @@ code to execute.::
 Questions
 ---------
 
-Find some uses cases where you'd use the if conditional.
+1. In an if conditional, what happens if the boolean is true?
+2. When does the `else` block of code run in an if else statement?
+3. What are some use cases for an if conditional?
```

Comment:
"use cases" is jargon, simplify to something more active e.g. "What could you use `if` for?"